scripts/obtain-plugin.sh

```diff
@@ -27,9 +27,6 @@ download-plugin() {
     chmod +x "$path"
 }
 
-for os in linux darwin
-do
-    download-plugin $os
-done
+download-plugin $(uname)
```

Comment:
This step is very stubbornly buggy - I get 400 Bad Request errors with no apparent rhyme or reason to when, I'll re-upload the same license and have that trigger a change. I was hoping changing this would fix that... but probably it didn't actually help.

Reply:
Reverted